sphinx/templates/latex/latex.tex_t

```diff
@@ -25,16 +25,6 @@
 \usepackage<%= sphinxpkgoptions %>{sphinx}
 <%= sphinxsetup %>
 <%= geometry %>
-\usepackage{multirow}
-\let\originalmutirow\multirow\protected\def\multirow{%
-  \sphinxdeprecationwarning{\multirow}{1.6}{1.7}
-  {Sphinx does not use package multirow. Its loading will be removed at 1.7.}%
-  \originalmultirow}%
-\usepackage{eqparbox}
-\let\originaleqparbox\eqparbox\protected\def\eqparbox{%
-  \sphinxdeprecationwarning{\eqparbox}{1.6}{1.7}
-  {Sphinx does not use package eqparbox. Its loading will be removed at 1.7.}%
-  \originaleqparbox}%
```

Comment:
Actually I think already at 1.6 we should not leave that in the latex template. If user needs `eqparbox`, it can load it. And if this happens after this, LaTeX will not reload it, so `\eqparbox` command will issue the deprecation warning, but the user deliberately wants to use it. I think this is bad. I don't think we can easily "deprecate" a package in LaTeX. And if an extension needs them it only has to do the `RequirePackage` itself. This is different from deprecating a macro which was originally defined by Sphinx and may be used by extensions.

what do you think @tk0miya about removing those already in 1.6-release branch ?